frontend/src/pages/SessionForm/__tests__/index.js

```diff
@@ -75,6 +77,7 @@ describe('SessionReportForm', () => {
   });
 
   it('fetches existing session report form', async () => {
+    jest.useFakeTimers();
```

Comment:
Why do  we need timers?

Reply:
The test was using them to test the autosave, this should have been in there.